nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -316,170 +345,172 @@ For example `/circuits/circuits/?provider_id=<uuid>` has been replaced by `/circ
+|                         | `parent_interface_id` |                                                                                               |
+|                         | `region`              |                                                                                               |
+|                         | `region_id`           |                                                                                               |
+|                         | `site`                |                                                                                               |
+|                         | `site_id`             |                                                                                               |
+| InventoryItem           | `device_id`           |                                                                                               |
+|                         | `manufacturer_id`     |                                                                                               |
+|                         | `parent_id`           |                                                                                               |
+|                         | `region`              |                                                                                               |
+|                         | `region_id`           |                                                                                               |
+|                         | `site`                |                                                                                               |
+|                         | `site_id`             |                                                                                               |
+| IPAddress               | `tenant_group_id`     |                                                                                               |
+| Location                | `tenant_group_id`     |                                                                                               |
+|                         | `region`              |                                                                                               |
+|                         | `region_id`           |                                                                                               |
+|                         | `site`                |                                                                                               |
+|                         | `site_id`             |                                                                                               |
+| Provider                | `region`              |                                                                                               |
+| ObjectPermission        | `user_id`             | instead of `/users/users/?user_id=<uuid>`, use `users=<uuid>`                                 |
```

Comment:
Text here looks wrong? Should this be `/users/object-permissions/` or something else?